### DISCUSSION TITLE
CLI exit delay

### DISCUSSION THREAD

**Discussion by vladchuk** [FilledStacks/stacked#1045] [General]
When analytics are enabled for stacked CLI, it causes a 15 seconds delay to exit the process after the command actually completes ("Command complete" output in the console).

I first thought of submitting a bug, but this just might be the cost of collecting analytics. **It it?..**
I don't mind sharing the usage, but the delay is quite annoying and so I generally disable it.

**Comment by FilledStacks** [comment]
We've also seen that, I don't think it's from the analytics, I actually don't know what causes it. 

When analytics is disabled it doesn't go away for me.
